docs/output.md

```diff
@@ -27,6 +47,247 @@ For further reading and documentation see the [FastQC help](http://www.bioinform
 * `zips/sample_fastqc.zip`
   * zip file containing the FastQC report, tab-delimited data file and plot images
 
+### Quality filtering and adaptor rrimming
+
+[Fastp](https://github.com/OpenGene/fastp) is a tool designed to provide fast all-in-one preprocessing for FastQ files. This tool is developed in C++ with multithreading supported to afford high performance.
+
+**Output directory: `preprocess/fastp`**
+* `sample.trim.fastq.gz`
```

Comment:
```suggestion
* `<SAMPLE>.trim.fastq.gz`
```

Reply:
Maybe use that notation throughout?

Reply:
Okay I'll replace the rest of the samples and finish polishing it a little bit more.